doc/manual/manual.tex

```diff
@@ -5945,36 +5945,32 @@ \subsubsection{Artificial viscosity smoothing}
+tensor $\mathbf E_{ij}$, and modifies the right-hand side of 
 the corresponding advection equations to accumulate strain over time. This is done by adjusting the 
 \verb!out.reaction_terms! variable:
 \lstinputlisting[language=C++]{cookbooks/finite_strain/finite_strain.cc}
 
-Let us denote the accumulated strain at time step $n$ as $e^n$. We can express it 
-as the sum of the strain $e^{n-1}$ at the previous time step, rotated by the rotational 
-component of the velocity field, plus the strain increment $\varepsilon(\mathbf u^n) \Delta t^n$ 
-accumulated over the current time step.
-Hence, the right-hand side term of the advection equation for the accumulated strain consists of two parts:
-The first one, $R e^{n-1} R^T$, rotates $e^{n-1}$, the accumulated strain from all the previous time steps;
-and the second part adds the strain of the current time step. 
+Let us denote the accumulated strain at time step $n$ as $\mathbf E^n$. We can calculate its time derivative  
+as the tensor product of the current velocity gradient $\mathbf G = \frac{\partial u_i}{\partial x_j}$ and the accumulated strain $\mathbf E^{n-1}$ at the previous time step. 
+While we refer to other studies \cite{McKenzie1983, dahlen1998theoretical, Becker2003} for a derivation of
+this relationship, we can give an intuitive example for the necessity to apply the velocity gradient to the already accumulated strain, instead of simply integrating the velocity gradient over time. Consider a simple one-dimensional ``grain'', where the strain tensor only has one component, the compression in x-direction. If one applies a convergent velocity gradient of 0.5 to it (e.g. a velocity of zero at its left end at $x=0.0$, and a velocity of $-0.5$ at $x=1.0$), integrating the velocity gradient would suggest that the grain reaches a length of zero after two units of time, and would then ``flip'' its orientation, which is clearly non-physical. Instead what happens is its length is cut in half in one unit of time, and continues two be halved per unit of time, which is identical to $\mathbf E^n = 0.5 \cdot \mathbf E^{n-1}$, or $\mathbf E^n = \mathbf G \cdot \mathbf E^{n-1}$.
+
+A direct visualization of $\mathbf E$ is not intuitive, because it contains rotational components that represent a rigid body rotation without deformation. Following \cite{Becker2003} we can polar-decompose the tensor into a positive-definite and symmetric left stretching tensor $\mathbf L$, and an orthogonal rotation tensor $\mathbf Q$, as $\mathbf E = \mathbf L \cdot \mathbf Q$, therefore $\mathbf L^2 = \mathbf E \cdot \mathbf E^T$. $\mathbf L$ describes the deformation we are interested in, and its eigenvalues $\lambda_i$ and eigenvectors $\vec{e}_i$ describe the length and orientation of the half-axes of the finite strain ellipsoid. Moreover, we will represent the amount of relative stretching at every point by the ratio $\ln(\lambda_1/\lambda_2)$, called the \textit{natural strain} \cite{Ribe1992}.
```

Comment:
I would omit the `\cdot`. It is used to denote the scalar product between two vectors, but is not necessary for two matrices.